examples/docker/from_anywhere/pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+runtime:
+  container_commands:
+    - "apt update -y"
+    - "apt install -y git"
+    # - "apt-get update && \
+    #   apt-get -y install gcc mono-mcs && \
+    #   rm -rf /var/lib/apt/lists/*"
+  python:
+    python_version: "3.10"
+    python_requirements:
+      - "pipeline-ai==1.0.26"
```

Comment:
May want to raise an error when the user forgets to provide a pipeline-ai version from now on?

Reply:
another option would be to use the version the user is currently using

Reply:
Do you mean if they have an activated environment, use the pipeline-ai version from that?

Reply:
yeah, I mean they have to run `pipeline container build` to build the container, so must have pipeline-ai installed, so we should be able to pick up the version directly